python/tk_desktop/loading_project_widget.py

```diff
@@ -107,14 +110,22 @@ def show_error_message(self, msg):
         :param str msg: Message to display
         """
         # Hide all widgets
-        self._ui.shotgun_spinning_widget.hide()
-        self._ui.bottom.hide()
-        self._ui.progress_output.hide()
+        self._show_hide_widgets(show=False)
 
         self.setVisible(True)
         self._message = msg
         self.repaint()
 
+    def _show_hide_widgets(self, show=False):
+        """
+        Shows or hides the widgets in the UI.
+
+        :param show: If True, shows all the widgets in the UI.
```

Comment:
I'd amend this just to make it clear that a `False` argument hides all the widgets in the UI.